index.html

```diff
@@ -25,10 +25,24 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
   </ul>
 
   <center>
-  <button v-on:click="swapToCompleted">Completed</button>
-  <button v-on:click="swapToActive">Active</button>
+  <button v-on:click="showCompleted">Completed</button>
+  <button v-on:click="showActive">Active</button>
 </div>
 
+<ul>
+    <li v-for="todo in activeTodos" :key="todo.id" class="flex">

```

Comment:
create another separate list for `<li v-for="todo in completedTodos" :key="todo.id" class="flex">`